lib/test/clojure_lsp/feature/move_form_test.clj

```diff
@@ -73,22 +73,22 @@
                      "(def bar foo)")
```

Comment:
@mainej this was the only failing test after the change, I took a quick look and it seems like a sorting issue after the change but not critical I guess, could you help confirm it please?

Reply:
I've never looked at the move-form code, but I'll do what I can. Will report back.

Reply:
From what I can see, there's code to pick from the known aliases, preferring aliases that are used the most. But, in this test, `bread` is aliased as `b` once and as `br` once. Since there's no tie breaker, they're equally valid choices. I'd suggest breaking the tie to make the code deterministic...

```diff
diff --git a/lib/src/clojure_lsp/feature/add_missing_libspec.clj b/lib/src/clojure_lsp/feature/add_missing_libspec.clj
index 81cc825b..e5a90ac5 100644
--- a/lib/src/clojure_lsp/feature/add_missing_libspec.clj
+++ b/lib/src/clojure_lsp/feature/add_missing_libspec.clj
@@ -312,8 +312,8 @@
                                                     (->> vs
                                                          (map first)
                                                          frequencies
-                                                         (sort-by val)
-                                                         reverse
+                                                         (sort-by (fn [[ns freq]]
+                                                                    [(- freq) ns]))
                                                          (into (array-map))))))
         namespaces->aliases (->> ns-alias-pairs
                                  (group-by first)
@@ -322,8 +322,8 @@
                                                          (map second)
                                                          (remove nil?)
                                                          frequencies
-                                                         (sort-by val)
-                                                         reverse
+                                                         (sort-by (fn [[alias freq]]
+                                                                    [(- freq) alias]))
                                                          (into (array-map))))))
         alias-namespaces (get aliases->namespaces cursor-namespace-str)
         namespace-aliases (get namespaces->aliases cursor-namespace-str)
```